maat-court-data-api/src/main/resources/db/changelog/changes/lasb_1718_grant_dart_read_permissions.sql

```diff
@@ -0,0 +1,22 @@
+-- We haven't included it here as we don't want a password being committed, but you need to create
+-- the user before running the script below.
+
+GRANT CREATE SESSION to DART_QUERY;
+commit;
+
+BEGIN
+    -- Add TOGDATA permissions
+    FOR t IN (SELECT table_name FROM dba_tables WHERE owner = 'TOGDATA') LOOP
+        EXECUTE IMMEDIATE 'GRANT SELECT ON TOGDATA.' || t.table_name || ' TO DART_QUERY';
+    END LOOP;
+
+    -- Add MLA permissions
+    FOR t IN (SELECT table_name FROM dba_tables WHERE owner = 'MLA') LOOP
+        EXECUTE IMMEDIATE 'GRANT SELECT ON MLA.' || t.table_name || ' TO DART_QUERY';
+    END LOOP;
+END;
+
```

Comment:
FYI did some reading around the "GRANT" command in Oracle - it's something called a Data Control Language (DCL) command and is automatically committed so _I don't think_ we need the commits in there for it.

Reply:
It's the same for the "REVOKE" command below.